src/cgame/cg_event.cpp

```diff
@@ -45,7 +45,7 @@ static const struct {
 	team_t   team;
 } meansOfDeath[] = {
 	// Icon            Envkill Assist? (Team)
-	{ "☠",             false, false, TEAM_HUMANS },
```

Comment:
Why is this team_humans?

I'm guessing this should  never happen?

Reply:
Should probably also be `NO_TEAM`. I only added the comment. I can add comments to all of them if you like, but they correspond to the list of `modNames` in https://github.com/Unvanquished/Unvanquished/blob/master/src/sgame/sg_combat.cpp

Reply:
Fixed in 400ae8a56d0be5f8b5d6cf7c4805c4971b9a4310